cmd/gaia/testnets/README.md

```diff
@@ -261,7 +230,7 @@ You should also be able to see your validator on the [Explorer](https://explorec
 
 ### Problem #1: My validator has `voting_power: 0`
 
-Your validator has become auto-unbonded. In `gaia-6002`, we unbond validators if they do not vote on `50` of the last `100` blocks. Since blocks are proposed every ~2 seconds, a validator unresponsive for ~100 seconds will become unbonded. This usually happens when your `gaiad` process crashes.
+Your validator has become auto-unbonded. In `gaia-7000`, we unbond validators if they do not vote on `50` of the last `100` blocks. Since blocks are proposed every ~2 seconds, a validator unresponsive for ~100 seconds will become unbonded. This usually happens when your `gaiad` process crashes.
```

Comment:
I think the unbonding period is longer in gaia-7000

Reply:
The signing window is 40,000 blocks (if you miss 20k out of 40k, you get unbonded).
The jail period is five minutes (time until you can unrevoke your validator after being unbonded for downtime).
The unbonding period is 24 hours.